client/http/http.go

```diff
@@ -29,10 +33,16 @@ func New(url string, chainHash []byte, transport nhttp.RoundTripper) (client.Cli
 	if !strings.HasSuffix(url, "/") {
 		url += "/"
 	}
+	pn, err := os.Executable()
+	if err != nil {
+		pn = defaultClientExec
+	}
+	agent := fmt.Sprintf("drand-client-%s/1.0", path.Base(pn))
```

Comment:
Shouldn't it use the actual version instead of 1.0 ? (per comment [here](https://github.com/filecoin-project/lotus/pull/4420#issuecomment-714011075))

Reply:
Actually, I see the other PR does a full overwrite. But still what does 1.0 mean.

Reply:
there isn't a way to know what version of the drand module has been loaded into another binary at build time. unless we defined a constant in the go source code that we update on each release it's going to be out of date with the drand version.
that's not something the go build system can help us with afaik

Reply:
1.0 here means this is the first version that's using this version string scheme 😆

Reply:
hmm you're right. :+1: